### PR TITLE
Add onboarding information

### PR DESCRIPTION
**Pull request by saundefined**
Added introductory information for new members.

Perhaps something needs to be corrected?

Relates #1148

### CODE REVIEW

**Comment by cmb69** [comment]
Thank you!

In my opinion, we should also link to http://doc.php.net/tutorial/, or maybe have multiple links to the relevant sections of that tutorial.

**Comment by salathe** [comment]
This is adding yet another copy of some "quick start" instructions that will not be kept up-to-date. :(

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+```
+├── doc-base/
+└── en/
+└── phd/
+```
+
+Note that language directory must be without the `doc-` prefix.
+
+For cloning, you can use the command `git clone https://github.com/php/doc-en.git en`.
```
Comment (nikic) [inline CONTRIBUTING.md:29]: FWIW, I would use less words, more copy&pastable commands :)
```
# Setup
git clone https://github.com/php/phd.git
git clone https://github.com/php/doc-base.git
git clone https://github.com/php/doc-en.git en

# Test your changes
php doc-base/configure.php
php phd/render.php --docbook doc-base/.manual.xml --package PHP --format xhtml
# Open output/php-chunked-xhtml/ in your browser.
```

**Comment by cmb69** [comment]
> This is adding yet another copy of some "quick start" instructions that will not be kept up-to-date. :(

Maybe we should remove that info from other places?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# PHP Manual Contribution Guide
```
Comment (salathe) [inline CONTRIBUTING.md:1]: This is missing a section on actually *contributing* changes.  There should probably be instructions on creating a fork (of doc-en, probably also doc-base), and adding the fork as a remote that contributors can push branches to, and creating PRs from their fork.

Comment (saundefined) [reply CONTRIBUTING.md]: Yes, I think it's definitely not worth pointing out here — there are more other guides on this.

Fixed the title with b201ee9.

Comment (salathe) [reply CONTRIBUTING.md]: So what's the point of this page if it's *not* going to be guiding folks on CONTRIBUTING(.md) to doc-en? 😕

Comment (saundefined) [reply CONTRIBUTING.md]: @salathe what do you think about this?
Just place links to the doc website, and "quick start" content?

Comment (afilina) [reply CONTRIBUTING.md]: I believe that this thread is waiting on a reply from @salathe.

Comment (afilina) [reply CONTRIBUTING.md]: @salathe Gentle nudge (see above). Does the syggestion by @saundefined make sense?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+````bash
+# Setup
+git clone https://github.com/php/phd.git
+git clone https://github.com/php/doc-base.git
+git clone https://github.com/php/doc-en.git en
+
+# Test your changes
+php doc-base/configure.php
+php phd/render.php --docbook doc-base/.manual.xml --package PHP --format xhtml
```
Comment (Girgias) [inline README.md:17]: I would make the output directory explicit with ``-o``

**Review by Girgias — APPROVED** [review]
Minor comment but LGTM

File: CONTRIBUTING.md (hunk 2 above)
Comment (salathe) [reply CONTRIBUTING.md]: I've said my piece already, and continue to not approve of the changes here.
That said, my approval isn't (and never was) required. 😊

**Pull request closed without merging**